app/helpers/application_helper.rb

```diff
@@ -1,2 +1,10 @@
 module ApplicationHelper
+    def full_title(page_title = '')
+        base_title = "Ruby on Rails Tutorial Sample App"
+        if page_title.empty?
+            base_title
+        else
+            page_title + " | " + base_title
```

Comment:
Normally something like this is extracted into its own service or helper. ... However its small and I guess can stay here 🤷🏻‍♂️ unless a lot more text stuff is added to the application_helper then you can extract it all at once.